src/Okta.Sdk/Internal/PkixCertPayloadHandler.cs

```diff
@@ -41,7 +41,17 @@ protected override HttpContent GetRequestHttpContent(HttpRequest httpRequest)
                 throw new ArgumentNullException("request payload");
             }
 
-            return new StringContent((string)httpRequest.Payload, Encoding.UTF8, ContentType);
+            var content = string.Empty;
+            if (httpRequest.Payload is string stringPayload)
+            {
+                content = stringPayload;
+            }
+            else if (httpRequest.Payload is byte[] bytePayload)
+            {
+                content = Convert.ToBase64String(bytePayload);
+            }
+
+            return new StringContent(content, Encoding.UTF8, ContentType);
```

Comment:
Because this payload handler can accept either base64 encoded string or byte array (byte[]), I had to account for that here.